pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ublue_updater"
+authors = [
+    {name = "gerblesh", email = "01901964+gerblesh@users.noreply.github.com"},
+]
+description = "Ublue System Updater"
```

Comment:
I'll probably do an overhaul of the docs later but we should try to switch away from uBlue and more towards the term "universal blue"